docs/rfcs/rfc-018-mathematical-operators.md

```diff
@@ -0,0 +1,167 @@
+# Mathematical operators
+
+## Problem
+As described in the issue #677, if we have to do a mathematical operation using the GraphQL library, 
+the common approach is to query the database and then perform the mutation with the final property value.
+The former approach has several downsides:
+- It requires multiple roundtrips to the database.
+- It's not concurrency safe.
+- It requires more code than necessary in the client.
+
+## Proposed Solution
+Add new input properties that solve atomic operations like:
+- `_INCREMENT`
+- `_DECREMENT`
+
+available for Int and BigInt fields.
+And:
+- `_ADD`
+- `_SUBTRACT`
+- `_MULTIPLY`
+- `_DIVIDE`
+
+available for Float fields.
+
+For instance `counter_ADD` or `counter_SUBTRACT`.
+These properties should be available for all the following types:
+- Int
```

Comment:
Just so I don't get it the wrong way around. If I read the text here in the proposed solution I get the understanding  that Int and BigInt should have the operators `_ADD`, `_SUBTRACT`, `_INCREMENT` and `_DECREMENT`?

Reply:
I meant that Int and BigInt fields have only _INCREMENT, _DECREMENT, and  Float fields have only _ADD, _SUBTRACT, _MULTIPLY, _DIVIDE.
Do you think that we can reword this differently to make it more clear?

Reply:
How about just removing these lines https://github.com/neo4j/graphql/pull/1353/files#diff-dfb3da75e39cfefef29c376e7e57172cbaa318b10748bb7174192cd9cb8d00dbR25-R29 then? Then it only contains the information you wrote in the comment above. Wdyt?

Reply:
It makes perfectly sense, thank you for the suggestion!